jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -492,6 +505,22 @@ private void deleteAllJobs() throws Exception {
 		}
 	}
 
+	private void uninitializeExecutors() throws MalformedObjectNameException {
+		this.uninitializeExecutors(queryExecutorRepository);
+		this.uninitializeExecutors(resultExecutorRepository);
+	}
+
+	private void uninitializeExecutors(ExecutorRepository executorRepository) {
```

Comment:
I'm not entirely sure that extracting `uninitializeExecutors(ExecutorRepository executorRepository)` added a lot of clarity, maybe inlining it would be better. (really minor, feel free to ignore)

Reply:
I was on the fence about this one but it seemed to make sense to have parity between the `initializeExecutors` and `uninitializeExecutors` methods. That way, if in the future there's some additional initialization happening, it'll be clear where to put the corresponding teardown code.